.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -162,7 +167,7 @@ jobs:
 
       - uses: actions/upload-artifact@v3
         if: |
-          github.event_name == 'release' ||
```

Comment:
Should we revert that before release?

Reply:
I think we need it to be `release` for the `download-artifact` action in the `upload-pypi` job.

Reply:
But here we are removing release and adding pull request. Right?

Reply:
Yes, sorry I misread your comment. This is reverted on  `v0.37.0_release`.

Reply:
This is not a 100% clear in the guidance docs, I'll update them accordingly.

Reply:
Nice!